content/docs/quickstart/go.md

```diff
@@ -141,26 +142,33 @@ message HelloReply {
 }
 ```
 
-### Generate gRPC code
+Remember to save the file!
+
+### Regenerate gRPC code
 
-Next we need to update the gRPC code used by our application to use the new
-service definition. From the same examples dir as above
-(`$GOPATH/src/google.golang.org/grpc/examples/helloworld`):
+Before you can use the new service method, you need to recompile the updated
+proto file.
+
+From the `examples/helloworld` directory, run the following command:
 
 ```sh
 $ protoc -I helloworld/ helloworld/helloworld.proto --go_out=plugins=grpc:helloworld
```

Comment:
@dfawley - It seems that source files fetched as modules are read only. I'm getting an error because the `.pb.go` file cannot be overwritten:

```console
$ protoc -I helloworld/ helloworld/helloworld.proto --go_out=plugins=grpc:helloworld
helloworld/helloworld.pb.go: Permission denied
```

Should I be fetching the example source in a different way?

Reply:
I think it would be best to copy the helloworld example to a new directory as part of these instructions (just the `.proto` and `.go` files?).

Reply:
@dfawley - Note that the client and server have imports like this:

```go
import ( ...
  pb "google.golang.org/grpc/examples/helloworld/helloworld"
)
```

So, if I copy the files to a local directory and create the corresponding module and try to run the server I get an "ambiguous import" error:

```console
$ cd "$HOME/examples/helloworld"
$ go mod init google.golang.org/grpc/examples/helloworld
$ go run greeter_server/main.go
greeter_server/main.go:30:2: ambiguous import: found package google.golang.org/grpc/examples/helloworld/helloworld in multiple modules:
        google.golang.org/grpc/examples/helloworld (/home/chalin/examples/hello_world/helloworld)
        google.golang.org/grpc v1.28.1 (/home/chalin/go/pkg/mod/google.golang.org/grpc@v1.28.1/examples/helloworld/helloworld)
```

Given https://github.com/golang/go/issues/32128, I believe that we don't currently have a way to resolve this ambiguity, right?

Reply:
I'm experimenting with creating a local module (for the copied sources) and changing the import paths. Does that sound like a sound approach? It seems to be working so far.

Reply:
@dfawley - done, PTAL